model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -40,8 +40,12 @@ class SimplePackageConfigurationProvider(
          * Return a [SimplePackageConfigurationProvider] which provides all [PackageConfiguration]s found recursively
          * in the given [directory]. All non-hidden files within the given [directory] must be package curation files.
          * Throws an exception if there is more than one configuration per [Identifier] and [Provenance].
+         * Additional configurations can be provided via [configurations].
          */
-        fun forDirectory(directory: File): SimplePackageConfigurationProvider {
+        fun forDirectory(
```

Comment:
Instead of adding a parameter, how about turning `forDirectory(): SimplePackageConfigurationProvider()` into
`readDirectory(): List<PackageConfiguration>`? ...For single responsibility.

Reply:
Would you make these functions top-level functions then as well? Because in the calling code `SimplePackageConfigurationProvider(SimplePackageConfigurationProvider.readDirectory(file) + configurations)` does not look so nice.

Reply:
That's probably a matter of taste. I'll leave it up to you.

Reply:
Done.